linode_api4/objects/lke.py

```diff
@@ -335,3 +407,40 @@ def service_token_delete(self):
         self._client.delete(
             "{}/servicetoken".format(LKECluster.api_endpoint), model=self
         )
+
+    def control_plane_acl_update(
```

Comment:
This pattern is not consistent with the rest of the codebase, but unfortunately we cannot implement it using the traditional properties because `ACL` is not returned from the LKE cluster GET endpoint.